runtime.md

```diff
@@ -33,24 +33,79 @@ This is provided so that consumers can find the container's configuration and ro
 }
 ```
 
-## Lifecycle
+## Operations
+
+OCI compliant runtimes MUST support the following operations, unless support for the operation can not be supported by the base operating system (kernel) itself.
+
+OPEN: we should consider specifying the options/args for each operation.
+For example, checkpoint (for runc) seems to have a lot of params.
```

Comment:
On Mon, Jan 04, 2016 at 10:44:32AM -0800, Doug Davis wrote:
> @wking can you remove the comments that aren't asking for a specific
> change?

Will do.  Now that we have a tagged set of open list issues [1,2]
(including the command-line API [3]), I think there's less need for
either my FYI comments or entries like this.  So my actionable
suggestion here is “drop this line in favor of [1] for listing open
high-level discussion”.

[1]: https://wking.github.io/nmbug-oci/
[2]: https://groups.google.com/a/opencontainers.org/forum/#!topic/dev/18TA4iOf48M
[3]: https://wking.github.io/nmbug-oci/#Features-with-external-work